corehq/motech/repeaters/views/repeaters.py

```diff
@@ -58,7 +58,7 @@ def repeaters(self):
                 r.__name__,
                 r.by_domain(self.domain),
                 r.friendly_name,
-                r.get_custom_url(self.domain)
+                r.has_config(),
```

Comment:
It took me quite a while to figure out how this was passing `has_config` into the HTML and then into the partial. What are your thoughts on using a `namedtuple` or `attrs` class instead of passing the tuple as `repeaters` and then as a `for` loop into the partial?

I'm not familiar enough with this code to tell if that's actually a good idea or not

Reply:
I think that's a good call. It'll make it a lot clearer.